code/__DEFINES/misc.dm

```diff
@@ -41,7 +42,7 @@
 #define BODY_FRONT_LAYER		3
 #define HALO_LAYER				2		//blood cult ascended halo, because there's currently no better solution for adding/removing
 #define FIRE_LAYER				1		//If you're on fire
-#define TOTAL_LAYERS			28		//KEEP THIS UP-TO-DATE OR SHIT WILL BREAK ;_;
+#define TOTAL_LAYERS			29		//KEEP THIS UP-TO-DATE OR SHIT WILL BREAK ;_;
```

Comment:
Maybe give an indication as to what has happened here

Reply:
ah yes i've yet to comment that part, i wasnt sure how as I moved a ton of layers